test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Import/ImportTests.cs

```diff
@@ -332,6 +334,94 @@ public async Task GivenIncrementalLoad_WhenOutOfOrder_ThenCurrentDatabaseVersion
             Assert.Equal(GetLastUpdated("2001"), result.Resource.Meta.LastUpdated); // version 1 imported
         }
 
+        [Fact(Skip = "requires new stored procedures")]
```

Comment:
Which new stored procedures? Is there a work item number that can be put here so we know when these can be enabled?

Reply:
Not now. It depends on results of future discussion on whether behaviors in the skipped tests are desirable or not.